01.md

```diff
@@ -1,177 +1,216 @@
 NIP-01
 ======
 
-Basic protocol flow description
--------------------------------
+Nostr Protocol
+--------------
 
 `draft` `mandatory`
 
-This NIP defines the basic protocol that should be implemented by everybody. New NIPs may add new optional (or mandatory) fields and messages and features to the structures and flows described here.
+This NIP defines the mandatory part of the Nostr protocol. New NIPs may add new optional (or mandatory) fields, messages, and features to the structures and flows described here.
 
-## Events and signatures
+# Events
```

Comment:
I always call these Notes instead of Events in my codebases, but maybe the ship has sailed on that one because of ["EVENT",...]

Reply:
"event" was probably a poor name choice, but "relay" too.

Reply:
I thought kind 1 events are "notes". All notes are events but not all events are notes.